aspnetcore/blazor/components/templated-components.md

```diff
@@ -20,49 +22,50 @@ Templated components are components that receive one or more UI templates as par
 
 * A table component that allows a user to specify templates for the table's header, rows, and footer.
 * A list component that allows a user to specify a template for rendering items in a list.
+* A nav-bar component that allows a user to specify a template for start content, and items.
```

Comment:
```suggestion
* A navigation bar component that allows a user to specify a template for start content and navigation links.
```

Reply:
I'm a little concerned about the "start content" language here ... that it isn't immediately clear what it mean. However, it's easy to see in the sample code what it means. Therefore, let's go with it. If someone has a problem with it, they can open an issue saying that they're confused by it.